action.yml

```diff
@@ -1,7 +1,23 @@
-name: "Action name"
-description: "Action description"
+name: "halt"
+description: |
+  Halt and un-halt open PRs by committing a file to your repository
 author: "Freckle"
 inputs:
+  default-branch:
+    description: "The branch on the halt file can be found"
+    require: true
+    default: "main"
+  halt-file:
+    description: "The path to the file used to halt"
+    require: true
+    default: ".github/HALT"
```

Comment:
Should this be in it's own directory? Are we risking bumping into other files if we build within the `.github` namespace?

Reply:
I considered it a mechanism for informing behavior on GitHub, and so `.github` made sense to me. I doubt they're going to implement something that uses a file named `HALT`, but yes it's technically true that that's a risk. Feel free to block and suggest an alternative, but I still like it as is.